js/views/itemEditVw.js

```diff
@@ -217,19 +220,33 @@ module.exports = Backbone.View.extend({
   },
 
   deleteImage: function(e) {
-    var imgIndex = $(e.target).closest('.itemImg').data('index');
+    var imageUploadArray, imgIndex = $(e.target).closest('.itemImg').data('index');
```

Comment:
I think it is a smart move. Forgive me if I am out of line. I think it is industry standard and a healthy practice. It prevents a lot of common JS errors. Maybe you'll appreciate John Resig's views on it. http://ejohn.org/blog/ecmascript-5-strict-mode-json-and-more/ Or Douglas Crockford views on it as it was directly inspired by his book Javascript: The Good Parts, as well as from the YUI team he headed: http://yuiblog.com/blog/2010/12/14/strict-mode-is-coming-to-town/